build/tasks/livereload.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
+var livereload = require( "tiny-lr" );
+var server;
+
+module.exports = function( grunt ) {
+	grunt.registerTask( "livereload", function() {
+		if ( !server ) {
+			server = livereload();
+			server.listen( 35729, function() {
```

Comment:
I left this out initially for lack of a solution that didn't demand a fixed port. Do you have any thoughts on that?